assets/js/base/components/shipping-calculator/address.js

```diff
@@ -19,17 +20,25 @@ const ShippingCalculatorAddress = ( {
 	addressFields,
 } ) => {
 	const [ address, setAddress ] = useState( initialAddress );
+	const { getValidationError } = useValidationContext();
+
+	const updateAddress = ( newAddress ) => {
+		setAddress( newAddress );
+	};
```

Comment:
Is this function needed anymore?

Reply:
Nope, good catch, we could just use `setAddress` directly.